VerifyEncoding/Test-Encoding.ps1

```diff
@@ -89,14 +100,33 @@ function Test-Encoding
                 continue
             }
 
+            $fileExists = Test-Path -Path $file
+
+            if ($fileExists -eq $False)
+            {
+                Write-Host "File $file is deleted. Skipping ..."
+                continue
+            }
+
+            $fileIsFolder = (Get-Item $file).PSIsContainer
+
+            if($fileIsFolder -eq $True)
```

Comment:
I prefer `!$fileExists` instead of `$fileExists -eq $False` and `$fileIsFolder` instead of `$fileIsFolder -eq $True`.